frontend/src/p2/forms/__tests__/WhatHappenedForm.test.js

```diff
@@ -40,8 +40,6 @@ describe('<WhatHappenedForm />', () => {
         </MockedProvider>
       </ThemeProvider>,
     )
-
-    const inputNode = getByLabelText('What happened?')
     const nextButton = getByText(/Continue/i)
 
     fillIn(inputNode, { with: 'lost money' })
```

Comment:
I think you may have to delete "fillIn(inputNode, { with: 'lost money' })" this line too, otherwise can't pass the test.